core-services/release-controller/_releases/release-ocp-4.9.json

```diff
@@ -24,5 +24,151 @@
     }}
   },
   "verify":{
+    "aws":{
+      "maxRetries": 3,
+      "prowJob":{"name":"periodic-ci-openshift-release-master-nightly-4.8-e2e-aws"}
```

Comment:
Does not seem right to use 4.8 here, how about we add 4.9 copies first?

Reply:
My bad.  Blind copy and paste.   Everything should be 4.9 now

Reply:
@petr-muller Is there a tool or anything to bump the jobs to a new version?

Reply:
Unfortunately not yet. I can do the work today.

Reply:
@petr-muller I attempted to add the jobs in a separate PR (#18903), to keep this on specifically for the release-controller.  Which is why I flagged this PR as WIP.  Unfortunately, the other PR's pj-rehearse jobs blew up in my face.  Not sure how to get that PR to progress or what type of voodoo I'm missing. :disappointed: